tuxedo-core/src/types.rs

```diff
@@ -132,6 +136,9 @@ pub enum UtxoError<ConstraintCheckerError> {
     VerifierError,
     /// One or more of the inputs required by this transaction is not present in the UTXO set
     MissingInput,
+    // TODO set a nicer description for this.
```

Comment:
Can complete this TODO or it still is lingering?

Reply:
```suggestion
```

Yeah, the existing one seems good to me.